package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "twentynineteen",
+  "version": "1.0.0",
+  "description": "Default WP Theme",
+  "bugs": {
+    "url": "https://github.com/WordPress/twentynineteen/issues"
+  },
+  "homepage": "https://github.com/WordPress/twentynineteen#readme",
+  "devDependencies": {
+    "autoprefixer": "^9.1.5",
+    "chokidar-cli": "^1.2.1",
+    "node-sass": "^4.9.3",
+    "postcss-cli": "^6.0.0",
+    "rtlcss": "^2.4.0"
+  },
+  "browserslist": [
+    "last 2 versions",
+    "ie >= 8"
+  ],
```

Comment:
The supported browsers should be the same as core, this is documented at:
> https://make.wordpress.org/core/handbook/best-practices/browser-support/

The ideal way of supporting the above is to use the `@wordpress/browserlist-config` package:
> https://github.com/WordPress/gutenberg/tree/master/packages/browserslist-config

To do this, run `npm install @wordpress/browserslist-config --save-dev` and then use:
```json
"browserslist": [
	"extends @wordpress/browserslist-config"
]
```